src/pages/en/reference/integrations-reference.md

```diff
@@ -194,15 +194,20 @@ A read-only copy of the user-supplied [Astro config](/en/reference/configuration
 A mutable instance of the Vite server used in "dev" and "preview" mode. For instance, this is [used by our Partytown integration](https://github.com/withastro/astro/tree/main/packages/integrations/partytown) to inject the Partytown server as middleware:
 
 ```js
-import
-
-'astro:server:setup': ({ server }) => {
-  server.middlewares.use(
-    partytownServer(partytownLibDirectory, {
-      mount: '/~partytown',
-      ...
-    })
-  );
+import partytown from '@astrojs/partytown'
+
```

Comment:
Changed my mind on the import sorry as we can’t actually import the middleware, it’s probably just misleading to show it.

```suggestion
```